trivia/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+        <link href="styles.css" rel="stylesheet">
+        <title>Trivia!</title>
+        <script>
+
+            // TODO Part 1
+            let IncorrectPartOne = false;
+
+            function change_content_part_one(txt){
+                document.getElementById('CorrectAnswer').innerHTML = txt;
+            }
+
+            function change_display_part_one(display) {
+                const AnswerAlert = document.getElementById("CorrectAnswer");
+
+                if(IncorrectPartOne == false) {
+                    change_content_part_one("Correct");
+                    AnswerAlert.style.color = "green";
+                }else {
+                    change_content_part_one("Incorrect");
```

Comment:
```suggestion
                if (IncorrectPartOne == false) {
                    change_content_part_one("Correct");
                    AnswerAlert.style.color = "green";
                } else {
                    change_content_part_one("Incorrect");
```

Check other places as well